<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSetup</class>
 <widget class="QDialog" name="GeneralSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="pathBox">
     <property name="title">
      <string>Default paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Listmode files</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="listfilepath"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="selectListpath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Histograms</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="histfilepath"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="selectHistpath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Config files</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="configfilepath"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="selectConfigpath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="idBox">
     <property name="title">
      <string>Run ID</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QSpinBox" name="runId">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>RUN ID will be transmitted in every data buffer header</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="runIdAuto">
        <property name="toolTip">
         <string>increment run ID automatically at every start cmd</string>
        </property>
        <property name="text">
         <string>Auto Incr.</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="runIdButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>set Run ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="histogramFileFormatBox">
     <property name="title">
      <string>Histogram file format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="standardHistogramFileFormat">
        <property name="toolTip">
         <string>Histogram data will be written in &quot;standard&quot; format</string>
        </property>
        <property name="text">
         <string>Standard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">histogramFileformatButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="simpleHistogramFileFormat">
        <property name="toolTip">
         <string>Histogram data file will be written in a &quot;simple&quot; format</string>
        </property>
        <property name="text">
         <string>Simple</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">histogramFileformatButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="writeProtectionBox">
     <property name="title">
      <string>Data file protections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="writeProtect">
        <property name="toolTip">
         <string>Remove writeable flag from data files</string>
        </property>
        <property name="text">
         <string>write protect data files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="tubeArrangementBox">
     <property name="title">
      <string>Arrangement of the tubes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="tube2D">
        <property name="toolTip">
         <string>Detector tubes are arranged in 2D</string>
        </property>
        <property name="text">
         <string>2D detector</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">arrangementButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="tube1D">
        <property name="toolTip">
         <string>Detector tubes are arranged in a &quot;line&quot;</string>
        </property>
        <property name="text">
         <string>1D detector</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">arrangementButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="triggerBox">
     <property name="title">
      <string>Trigger input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QComboBox" name="triggerInput">
        <property name="toolTip">
         <string>Select the trigger input for time resolved measurements</string>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monitor 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monitor 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monitor 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monitor 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TTL 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TTL 2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectListpath</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>selectListpathSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectHistpath</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>selectHistpathSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectConfigpath</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>selectConfigpathSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runIdButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>setRunIdSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runIdAuto</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>selectAutoIncRunIdSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeProtect</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSetup</receiver>
   <slot>selectWriteProtectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectListpathSlot()</slot>
  <slot>selectConfigpathSlot()</slot>
  <slot>selectHistpathSlot()</slot>
  <slot>setRunIdSlot()</slot>
  <slot>selectAutoIncRunIdSlot()</slot>
  <slot>selectWriteProtectSlot()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="arrangementButtonGroup"/>
  <buttongroup name="histogramFileformatButtonGroup"/>
 </buttongroups>
</ui>
